Assets/Plugins/Source/Editor/Configs/LibraryBuildConfig.cs

```diff
@@ -21,10 +21,19 @@
  */
 namespace PlayEveryWare.EpicOnlineServices.Editor.Config
 {
+    using System;
+
+    [Serializable]
+    [ConfigGroup("Platform Library Build Settings")]
     public class LibraryBuildConfig : EditorConfig
     {
+        [FilePathField("MSBuild path", "exe")]
         public string msbuildPath;
+
+        [FilePathField("Make path", "")]
```

Comment:
No extension for Linux, but it _does_ support having make on windows, so this should probably be `*`.